### ISSUE TITLE
libplacebo configure failed because not found "dlltool"

### ISSUE THREAD

**Issue by zhongfly** [shinchiro/mpv-winbuild-cmake#363]
log:
```
Program llvm-dlltool dlltool found: NO

../../../../../src_packages/libplacebo/src/meson.build:23:12: ERROR: Program 'llvm-dlltool dlltool' not found or not executable
```
https://github.com/haasn/libplacebo/commit/34a70f9756d8fa28ec3533aac37f1b032fa51dec require the 'llvm-dlltool'

**Comment by shinchiro** [comment]
Should complain in upstream as there's no llvm-* in gcc+binutils toolchain

**Comment by shinchiro** [comment]
Should be fixed with [3fe50c0](https://github.com/shinchiro/mpv-winbuild-cmake/commit/3fe50c05f7cfb99e2e74a81cef9486a34e28d4bb)
